public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -266,6 +278,46 @@ export default class FindingDetailsFlyout extends Component<
     );
   }
 
+  createIndexPatternModal() {
+    const {
+      finding: { related_doc_ids },
+    } = this.props;
+    if (this.state.isCreateIndexPatternModalVisible) {
+      return (
+        <EuiModal
+          style={{ width: 800 }}
+          onClose={() => this.setState({ ...this.state, isCreateIndexPatternModalVisible: false })}
+        >
+          <EuiModalHeader>
+            <EuiModalHeaderTitle>
+              <h1>Create index pattern to view documents</h1>
+            </EuiModalHeaderTitle>
+          </EuiModalHeader>
+
+          <EuiModalBody>
+            <CreateIndexPatternForm
+              indexPatternsService={this.props.indexPatternsService}
+              initialValue={{
+                name: this.props.finding.detector._source.inputs[0].detector_input.indices[0] + '*',
+              }}
+              close={() =>
+                this.setState({ ...this.state, isCreateIndexPatternModalVisible: false })
+              }
+              created={(indexPatternId) => {
+                this.setState({
+                  ...this.state,
+                  indexPatternId,
+                  isCreateIndexPatternModalVisible: false,
+                });
+                window.open(`discover#/context/${indexPatternId}/${related_doc_ids[0]}`, '_blank');
```

Comment:
Similarly here, let's use history object

Reply:
I was looking for it, but based on my research (docs, google and chatGPT 😆) its not possible to open in new tab via react-router-dom